pkg/config/number_with_suffix.go

```diff
@@ -22,20 +22,20 @@ import (
 	"strings"
 )
 
-type Replicas struct {
+type NumberWithSuffix struct {
```

Comment:
How about: `UnitSuffixableNumber`? 🤔

Reply:
to be honest, it looks too "heavy" to me 😂